waku/node/peer_manager/waku_peer_store.nim

```diff
@@ -140,8 +145,10 @@ proc getPeersByProtocol*(peerStore: PeerStore, proto: string): seq[RemotePeerInf
   return peerStore.peers.filterIt(it.protocols.contains(proto))
 
 proc getReachablePeers*(peerStore: PeerStore): seq[RemotePeerInfo] =
+  let threshold = Moment.fromNow(millis(-3*60*60*1000))
```

Comment:
Are we sure the units are in seconds?

Reply:
Actually, the secs and hours functions cause compilation errors. will try to move hourly formula. regarding 3 hours I will check with alvaro before merging.